ocs_ci/deployment/deployment.py

```diff
@@ -491,6 +491,17 @@ def deploy_ocs_via_operator(self, image=None):
 
         cluster_data["metadata"]["name"] = config.ENV_DATA["storage_cluster_name"]
 
+        # Flexible scaling is available from version 4.7
+        if config.ENV_DATA.get("enable_flexible_scaling") and config.DEPLOYMENT.get(
+            "local_storage"
+        ):
+            ocs_version = float(config.ENV_DATA["ocs_version"])
```

Comment:
If this enable_flexible_scaling should be really enabled by default for each LSO cluster on version >= 4.7 we should change condition to something like this:
```suggestion
        ocs_version = float(config.ENV_DATA["ocs_version"])
        if config.ENV_DATA.get("enable_flexible_scaling") or (
            config.DEPLOYMENT.get("local_storage") and ocs_version >= 4.7)
        ):
```